EIPS/eip-7406.md

```diff
@@ -0,0 +1,155 @@
+---
+eip: 7406
+title: Multi-Namespace Onchain Registry
+description: An universally accepted multi-namespace registry with mapping structures on the Ethereum
+author: Mengshi Zhang (@MengshiZhang), Zihao Chen (@zihaoccc)
+discussions-to: https://ethereum-magicians.org/t/eip-7406-multi-namespace-onchain-registry/15216
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
```suggestion
category: ERC
```
This is an ERC (application layer)

Reply:
thanks for reviewing, resolved

Reply:
@abcoathup Hi Andrew, may I ask for your kind review or any feedback when you have time. Much appreciated